packages/fast-components-react-msft/src/badge/badge.spec.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from "react";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, mount, shallow } from "enzyme";
+import { BadgeClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import MSFTBadge from "./badge";
+import {
+    Badge,
+    BadgeAppearance,
+    BadgeHandledProps,
+    BadgeProps,
+    BadgeSize,
+    BadgeUnhandledProps,
+} from "./index";
+
+/*
+ * Configure Enzyme
+ */
+configure({ adapter: new Adapter() });
+
+describe("badge", (): void => {
+    const managedClasses: BadgeClassNameContract = {
+        badge: "badge",
+        badge__small: "badge__small",
+        badge__large: "badge__large",
+        badge__lowlight: "badge__lowlight",
+        badge__highlight: "badge__highlight",
+        badge__accent: "badge__accent",
+    };
+
+    test("should have a displayName that matches the component name", () => {
+        expect((MSFTBadge as any).name).toBe(MSFTBadge.displayName);
+    });
+
+    test("should not throw if managedClasses are not provided", () => {
+        expect(() => {
+            shallow(<MSFTBadge />);
+            shallow(<MSFTBadge appearance={BadgeAppearance.lowlight} />);
+            shallow(<MSFTBadge appearance={BadgeAppearance.highlight} />);
+            shallow(<MSFTBadge appearance={BadgeAppearance.lightweight} />);
```

Comment:
Looks like we can remove lightweight from here

Reply:
See other comment.

Reply:
Lightweight removed